test/hotspot/jtreg/compiler/profiling/spectrapredefineclass_classloaders/Launcher.java

```diff
@@ -65,7 +66,7 @@ public static void main(String[] args) throws Exception  {
 
         ProcessBuilder pb = new ProcessBuilder(jar.getCommand());
         try {
-            OutputAnalyzer output = new OutputAnalyzer(pb.start());
+            OutputAnalyzer output = ProcessTools.executeProcess(pb);
             output.shouldHaveExitValue(0);
         } catch (IOException ex) {
```

Comment:
Can probably also be updated as in Test7068051?
```suggestion
        } catch (Exception ex) {
```